docs/servers.rst

```diff
@@ -135,29 +135,25 @@ following output in your terminal. ::
 Create the Ubuntu Installation Media
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To create the Ubuntu installation media, you can either burn the ISO image to a
-CD-R or create a bootable USB stick.  The ``dd`` command can be used to copy the
-hybrid ISO directly to a USB drive, instead of a utility like UNetbootin which
-can result in errors. Once you have a CD or USB with an ISO image of Ubuntu on
-it, you may begin the Ubuntu installation on both SecureDrop servers.
-
-To use `dd` you first need to find where the USB device you wish to install
-Ubuntu on has been mapped. Simply running the command ``lsblk`` in the terminal
-will give you a list of your block storage device mappings (this includes hard
-drives and USB). If the USB you are writing the Ubuntu installer to is of a
-different size or brand than the USB you are running Tails from, it should be
-easy to identify which USB has which sdX identifier. If you are unsure, try
-running ``lsblk`` before and after plugging in the USB you are using for the
-Ubuntu installer. Note that you should use the main block device (e.g. ``/dev/sdb``)
-rather than any listed partitions (e.g. ``/dev/sdb2``).
-
-If your USB is mapped to /dev/sdX and you are currently in the directory that
-contains the Ubuntu ISO, you would use dd like so: ::
-
-   sudo dd conv=fdatasync if=ubuntu-20.04.4-live-server-amd64.iso of=/dev/sdX
+To create the Ubuntu installation media on a bootable USB stick, open the
+**Disks** program (**Applications ▸ Utilities ▸ Disks**) used earlier.
+
+Select the USB device you would like to use and then open the menu in the top
+right, selecting **Restore Disk Image**.
```

Comment:
As discussed in sprint planning yesterday, it may be appropriate to add a brief reminder here that this list includes all disks, so it's important to ensure that the selection is correct to avoid accidentally overwriting data.

Reply:
Done.